src/main/java/com/example/backendspringboottechiteasycontrollerles10/dtos/remoteControls/RemoteControlDto.java

```diff
@@ -0,0 +1,94 @@
+package com.example.backendspringboottechiteasycontrollerles10.dtos.remoteControls;
+
+import com.example.backendspringboottechiteasycontrollerles10.dtos.televisions.TelevisionDto;
+import jakarta.validation.Valid;
+
+@Valid
```

Comment:
Deze annotatie hoort voor de parameter in de Controller (dus de @Valid @RequestBody), maar daar gebruik je een RemoteControlIinputDto